zeppelin-web/src/app/helium/helium.controller.js

```diff
@@ -24,6 +25,16 @@ function HeliumCtrl($scope, $rootScope, $sce, baseUrlSrv, ngToast, heliumService
   $scope.showVersions = {};
   $scope.bundleOrder = [];
   $scope.bundleOrderChanged = false;
+  $scope.vizTypePkg = {}
+  $scope.spellTypePkg = {}
+  $scope.intpTypePkg = {}
+  $scope.appTypePkg = {}
+  $scope.numberOfEachPackageByType = {}
+  
+  $scope.allPackageTypes = [HeliumType][0]
+  $scope.pkgListByType = 'VISUALIZATION'
```

Comment:
@1ambda Thanks for your suggestion. I created an Jira ticket for this [ZEPPELIN-2209](https://issues.apache.org/jira/browse/ZEPPELIN-2209). Will start work after this PR merged then :)